tensorflow/lite/tools/benchmark/README.md

```diff
@@ -87,6 +87,13 @@ and the following optional parameters:
     `enable_op_profiling`. When this is set to true the profile of ops
     on hexagon DSP will be added to the profile table.
     Note that, the reported data on hexagon is in cycles, not in ms like on cpu.
+*   `use_external_delegate`: `bool` (default=false) \
+    Whether to use a specified external delegate library.
+*   `external_delegate_path`: `string` (default="") \
+    Path to the external delegate library.
+*   `external_delegate_options`: `string` (default="") \
+    A list of options to be passed to the external delegate library.
+    Options should be in the format of `option1:value1,option2:value2,optionN:valueN`
```

Comment:
I'm thinking it would be better to use `;` for the separator rather than `,`?

Reply:
Yes I agree with you. Will switch Jared to using `;`